cmd/internal/ec2/add.go

```diff
@@ -269,6 +272,26 @@ func enablePostmanInsightsAgent() error {
 	if err != nil {
 		return errors.Wrapf(err, "failed to run systemctl daemon-reload")
 	}
+
+	// systemctl check if postman-insights-agent.service is active
+	cmd = exec.Command("systemctl", []string{"is-active", serviceFileName}...)
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		printer.Infof("Postman Insights Agent is not running as a systemd service, status: %v\n", err)
+	} else if strings.Contains(string(output), active) {
+		printer.Infof("Postman Insights Agent is already running as a systemd service\n")
+		printer.Infof("Restarting the service with new configurations\n")
+
+		// systemctl restart postman-insights-agent.service
+		cmd = exec.Command("systemctl", []string{"restart", serviceFileName}...)
+		_, err = cmd.CombinedOutput()
```

Comment:
Can you clarify whether it was intended that, if the agent is already running, that we both "restart" and "enable --now"?  I kind of expected we would do one or the other, but not both.  If intended, maybe leave a comment here to explain why.